src/Modules/Finance/Header/Total/Total.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import styles from '../Form.module.css';
+
+interface ITotalProps {
+	value: string
+}
+
+export const Total: React.FC<ITotalProps> = (props: ITotalProps) => {
+	return (
+		<div className={styles.Form}>
+			<h3 className={styles.Title}>Total</h3>
+			<input className={styles.Input} type="text" value={props.value} />

```

Comment:
It should be an ordinary `text` field, not input, as user is not supposed to change it manually.

Reply:
As I wrote yesterday, all the code related to Header and its children appeared accidentially in this branch. I'll push a new branch, dedicated to Header, later. The "Total" component is just a copy/pasted from "TargetSum".

Reply:
will be converted in the bug